README.md

```diff
@@ -1822,6 +1822,22 @@ get do
 end
 ```
 
+The above example may have been requested as follows:
+
+``` shell
+curl -H "secret_PassWord: swordfish" ...
+```
+
+The header name will have been normalized for you.
+
+- In the `header` helper names will be coerced into a capitalized kebab case.
```

Comment:
kebab case?

Reply:
It's that thing where you use '-' between all words so they look like they're on a kebab.  https://medium.com/@pddivine/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
Oh 🍖